builtin/providers/postgresql/provider.go

```diff
@@ -35,6 +35,12 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("POSTGRESQL_PASSWORD", nil),
 				Description: "Password for postgresql server connection",
 			},
+			"sslmode": &schema.Schema{
```

Comment:
Normally we separate words with underscores, so I'd expect this to be `ssl_mode`.

However, we do make exceptions for things that are expected to be familiar to users of the service a provider is interacting with. So we could leave it as `sslmode` if that's how it's named in some visible part of the Postgres "UI", which I think it might be here because it seems like `connStr` is something like a DSN that you'd specify when configuring a client. Is that right?